src/mono/sample/wasi/console/Wasi.Console.Sample.csproj

```diff
@@ -4,6 +4,7 @@
     <!-- 
     <WasmSingleFileBundle>true</WasmSingleFileBundle>
     <InvariantGlobalization>true</InvariantGlobalization>
+    <_WasiDebugger>true</_WasiDebugger>
```

Comment:
Please, mention that it works only in in-tree sample as it is implemented in `Directory.Build.targets`